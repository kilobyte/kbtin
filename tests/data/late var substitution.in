#var x A1
#alias foo {#alias bar {#showme $x}}
foo
bar
#var x B1
bar
#var x C1
foo
#var x D1
bar
#var x A2
#alias foo {#alias bar {#showme $$x}}
foo
bar
#var x B2
bar
#var x C2
foo
#var x D2
bar
#var x A3
#alias foo {#alias bar {#showme $$$x}}
foo
bar
#var x B3
bar
#var x C3
foo
#var x D3
bar
