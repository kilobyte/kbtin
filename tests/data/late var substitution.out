A1
B1
D1
A2
A2
C2
A3
A3
A3
